### PR TITLE
Update 4 secure

### PR DESCRIPTION
**Pull request by qiwzhang**
Also changed release-publish to publish the secure image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,25 @@ To download the Extensible Service Proxy source code, clone the ESP repository:
...
+
+ESP docker images are released regularly. The regular images are named as gcr.io/endpoints-release/endpoints-runtime:MAJOR_VERSION.MINOR_VERSION.PATCH_NUMBER. For example, gcr.io/endpoints-release/endpoints-runtime:1.30.0 has MAJOR_VERSION=1, MINOR_VERSION=30 AND PATCH_NUMBER=0.
+
+Symbolically linked images:
+* **MAJOR_VERSION** is linked to the latest image with same **MAJOR_VERSION**.
+
+For example, gcr.io/endpoints-release/endpoints-runtime:1 is always pointed to the latest image with "1" major version.
+
+### Secure image: ###
+Normally ESP container runs as root, it is deemed as not secure. To make ESP container secure, it should be run as non-root and its root file system should be read-only. Normal docker images can be made to run as non-root, but such change may break some existing users. Starting 1.31.0, a new **secure image** is build with suffix "-secure" in the image name, e.g. gcr.io/endpoints-release/endpoints-runtime-secure:1.31.0.  It will be run as non-root.
```
Comment (kh-chang) [inline README.md]: nit: ... is `built` with suffix "-secure" ...

File: README.md [hunk 2]
```diff
@@ -45,6 +45,25 @@ To download the Extensible Service Proxy source code, clone the ESP repository:
     # Initialize Git submodules.
     git -C esp submodule update --init --recursive
 
+## Released ESP docker images ##
+
+ESP docker images are released regularly. The regular images are named as gcr.io/endpoints-release/endpoints-runtime:MAJOR_VERSION.MINOR_VERSION.PATCH_NUMBER. For example, gcr.io/endpoints-release/endpoints-runtime:1.30.0 has MAJOR_VERSION=1, MINOR_VERSION=30 AND PATCH_NUMBER=0.
```
Comment (kh-chang) [inline README.md]: nit: AND -> and

File: README.md [hunk 3]
```diff
@@ -45,6 +45,25 @@ To download the Extensible Service Proxy source code, clone the ESP repository:
...
+
+Symbolically linked images:
+* **MAJOR_VERSION** is linked to the latest image with same **MAJOR_VERSION**.
+
+For example, gcr.io/endpoints-release/endpoints-runtime:1 is always pointed to the latest image with "1" major version.
+
+### Secure image: ###
+Normally ESP container runs as root, it is deemed as not secure. To make ESP container secure, it should be run as non-root and its root file system should be read-only. Normal docker images can be made to run as non-root, but such change may break some existing users. Starting 1.31.0, a new **secure image** is build with suffix "-secure" in the image name, e.g. gcr.io/endpoints-release/endpoints-runtime-secure:1.31.0.  It will be run as non-root.
+
+You can switch to use the **secure images** if statisfied followings:
```
Comment (kh-chang) [inline README.md]: nit: "if the followings are satisfied:"

Comment (qiwzhang) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (qiwzhang) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (qiwzhang) [reply README.md]: Done

**Pull request merged**